queue/queue.py

```diff
@@ -1,17 +1,38 @@
 import sys
 sys.path.append('../linked_list')
-from linked_list import LinkedList
+from linked_list import LinkedList, Node
 
```

Comment:
Adding `Node` is not necessary; only `LinkedList` depends on `Node`, and that is already imported in `linked_list.py`.